eventsub.go

```diff
@@ -0,0 +1,418 @@
+package helix
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// EventSub Types for Parsing Requests / Responses
+
+// Represents a subscription
+type EventSubSubscription struct {
+	ID		  string			`json:"id"`
+	Type      string            `json:"type"`
+	Version   string            `json:"version"`
+	Status	  string            `json:"status"`
+	Condition EventSubCondition `json:"condition"`
+	Transport EventSubTransport `json:"transport"`
+	CreatedAt string            `json:"created_at"`
```

Comment:
The type here should be `Time` rather than `string`. See https://github.com/nicklaw5/helix/blob/master/time.go#L16-L37

```suggestion
	CreatedAt Time            `json:"created_at"`
```